charts/redis/templates/redis-service.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.managed_redis.disabled }}
```

Comment:
Can we rename this file to `Service.yaml` ?

Reply:
Yup, It's updated